schemas/dbt_project.json

```diff
@@ -139,6 +139,18 @@
                 }
             ]
         },
+        "label_configs": {
+            "title": "Label configs",
+            "type": "object",
+            "description": "Configurations specific to Big Query adapter used to add labels and tags to tables & views created by dbt.",
+            "patternProperties": {
```

Comment:
This is a really nice use of patternProperties 🤩

Reply:
Thanks 🙏